index.mjs

```diff
@@ -378,9 +431,24 @@ export default function runTest262({ test262Dir, testGlobs, polyfillCodeFile, ex
     }
     for (const [expectedFailureFile, unexpectedPassesSet] of unexpectedPasses) {
       if (updateExpectedFailureFiles) updateExpectedFailureFile(expectedFailureFile, unexpectedPassesSet);
-      print(` \u2022  ${expectedFailureFile}:`);
+      print(` \u2022 ${expectedFailureFile}:`);
       for (const unexpectedPass of unexpectedPassesSet) {
-        print(`${unexpectedPass}`);
+        print(`   \u2022 ${unexpectedPass}`);
+      }
+    }
+  }
+
+  if (missingTestsMap.size > 0) {
+    if (updateExpectedFailureFiles) {
+      print(`\n${color.yellow.bold('WARNING:')} Test files missing; the following tests have been removed from their respective files:`);
```

Comment:
"from their respective files"

Is this referring to the expected-failure file that contains that test name? If so, can this be more explicitly mentioned?

Reply:
Thanks for the feedback @12wrigja. I made the text clearer.